UnitTests/Schemas/IFC4X1/air-terminal-element/new 1

```diff
@@ -0,0 +1,20 @@
+#
+#    Copyright (c) 2020 Technical University of Munich
+#    Chair of Computational Modeling and Simulation.
+#
+#    TUM Open Infra Platform is free software; you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License Version 3
+#    as published by the Free Software Foundation.
+#
+#    TUM Open Infra Platform is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+include(CreateUnitTests)
+
+CreateIfcFileUnitTestForSchema(air-terminal-element IFC4X1)
```

Comment:
What is this file used for?

(I'm commenting the last line of the file in question)